__tests__/useActivity.test.tsx

```diff
@@ -6,7 +6,8 @@ import useActivity, {
   FullActivityDTO,
   useSingleActivity,
 } from "../hooks/useActivity";
-import CitizenProvider from "../providers/CitizenProvider";
+import WeekplanProvider from "../providers/WeekplanProvider";
+import { createActivityCitizen } from "../apis/activityAPI";
```

Comment:
Check warning on line 10 in __tests__/useActivity.test.tsx
'createActivityCitizen' is defined but never used